### ISSUE TITLE
Review and estimate remaining work before publication

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#597]
Today @kathy-phet recommended I survey outstanding work for CCK-AC to see what is left to do before publication.  We would like to publish this sim by January.   Working backwards, that means RC Dec 1 and Dev version Nov 1.  So I will need to start this sometime in Sept.

**Comment by arouinfar** [comment]
@samreid I'm going to co-assign myself to have a look through open issues and phethelp emails.

**Comment by samreid** [comment]
I estimate less than an hour to incorporate the new icons, a day to address TODOs in the code and 2-4 days to address the issues linked in https://github.com/phetsims/circuit-construction-kit-common/issues/528.  Those are all the issues assigned to me at the moment (could save time if we skip a tricky one).  But there are 18 other issues spanning new features, design changes, etc. which would be good to know more about before I estimate timing.

**Comment by samreid** [comment]
I wrote to @kathy-phet on slack:

I cannot easily give a “bottom line” timeline, but let me summarize what I know at the moment.
There are 6 known issues for the “DC” milestone: https://github.com/phetsims/circuit-construction-kit-common/milestone/1
None of those is very complicated.  The big delays in going toward DC publication are (a) waiting on art assets (b) waiting on QA and (c) any redesign or major changes.  This will have a long QA cycle.

There are 15 open issues for the “AC” milestone: https://github.com/phetsims/circuit-construction-kit-common/milestone/2. Several of these are medium-difficult complexity, and will require several iterations of feedback/design/meetings.  This will have an even longer QA cycle.  I predict that there will be several design changes or bug discoveries that we don’t know about at the moment, and they will be nontrivial.

So without rushing any part of the process, it feels like AC RC could start testing sometime in February.  That is a very rough guess, and if we reprioritize parts for the team, or postpone some feature requests we could move that up.
